docs/Contributing.md

```diff
@@ -126,6 +126,10 @@ When implementing a new blockchain make sure you go through this checklist:
 
 Also check out the [Adding Support for a New Blockchain](https://github.com/TrustWallet/wallet-core/wiki/Adding-Support-for-a-New-Blockchain) document.
 
+## Code Style
+
+Wallet core follows the [LLVM Coding Standards](http://llvm.org/docs/CodingStandards.html) for C++. We use `clang-format` to ensure a consistent code sytle. Please run `git clang-format` on your code before you submit a PR. **Do not** reformat files that you didn't modify. You can also install a clang-format extension for your IDE.
```

Comment:
is there a way to run this on CI, similar to `danger` or somehow automatic? I'm pretty sure most would forget to run this

Reply:
Yes but there is no point because what it does is format the code. It would need to have push access to the author's fork.